### PR TITLE
build: add developer field to parent pom

### PR DESCRIPTION
**Pull request by Neenu1995**

### CODE REVIEW

**Review by elharo — APPROVED** [review]

**Comment by elharo** [comment]
:      Non-parseable POM /home/runner/work/google-api-java-client/google-api-java-client/pom.xml: Duplicated tag: 'organization' (position: START_TAG seen ...</url>\n  </scm>\n\n  <organization>... @43:17)  @ line 43, column 17 -> [Help 2]

File: pom.xml [hunk 1]
```diff
@@ -17,6 +17,22 @@
...
+      <id>chingor</id>
+      <name>Jeff Ching</name>
+      <email>chingor@google.com</email>
+      <organization>Google</organization>
+      <roles>
+        <role>Developer</role>
+      </roles>
+    </developer>
+  </developers>
+  <organization>
```
Comment (elharo) [inline pom.xml:32]: remove this. There's already an organization on line 27

Comment (Neenu1995) [reply pom.xml]: removed

**Review by chingor13 — APPROVED** [review]
I'd probably mark this as a `build:` commit

**Pull request merged**